keyboards/dztech/dz65rgb/v3/rules.mk

```diff
@@ -0,0 +1,28 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = lufa-ms
```

Comment:
I see you have changed this back. Have you tested it with firmware files over 26,624 bytes? Currently the `BOOTLOADER_SIZE` for `lufa-ms` is incorrect - LUFA has increased the size from 6kB to 8kB, so when using the `RESET` keycode etc. it will be jumping to the wrong place.

Reply:
I tested this with different sizes.
Using an oversized file just leads to the board not starting up. I can be put back into bootloader mode afterwards just fine.

The `RESET` keycode doesn't seem to care about the size of the lufa-ms bootloader in QMK (afaik QMK doesn't have any reset magic for that bootloader anyways?).
If the fuses are set correct the `RESET` key worked just fine on the 8k bootloader as well.

Reply:
> The RESET keycode doesn't seem to care about the size of the lufa-ms bootloader in QMK

Yes, it does. QMK needs to know the address to jump to when `bootloader_jump()` is called: https://github.com/qmk/qmk_firmware/blob/master/tmk_core/common/avr/bootloader.c#L287
Since it is set to 6144 bytes at the moment, it will jump to the middle of the aux bootloader section, instead of the trampoline or the boot section start.

Reply:
ok.

I already opened a PR for updating the Lufa-MS size.

Drashna requested to not change it directly in bootloader.mk though and instead make it configurable and add the 8k as an override to the boards that use the new bootloader until Lufa in QMK gets updated.

I guess we should add a `BOOTLOADER_SIZE = 8192` to the rules.mk on this then.

Reply:
> until Lufa in QMK gets updated.

No need, it's been in since the last LUFA submodule update: https://github.com/qmk/lufa/blob/HEAD/Bootloaders/MassStorage/makefile#L31

Reply:
so going the other way round?

make it configurable but override it in the old board back to 6k (like Tada68)?

Reply:
Yep, and with a note along the lines of "this board's bootloader is an older version of LUFA-MS that is prone to bricking but slightly smaller".